nservicebus/containers/index.md

```diff
@@ -16,6 +20,38 @@ NServiceBus automatically registers all its components as well as user-implement
 
 NServiceBus has a built-in container (currently an ILMerged version of Autofac) but it can be replaced by any other container.
 
+Using an external container is usefull when:
+
+* A external container is already present and you want types that are registered in that container to be injected in your messages handlers or other types that NServiceBus creates internally.
+* Have lifetime requirements on the created instances that are not supported by the  NServiceBus object builder.
+* Not want to use the NServiceBus object builder because you are already familiar with other dependency injection frameworks.
```

Comment:
reading the starting statement with the second two points does not flow. for example

> Using an external container is usefull when have lifetime requirements

> Using an external container is usefull when Not want to use the NServiceBus

and usefull  is a typo